Manifests/ManifestsApple/com.apple.security.acme.plist

```diff
@@ -13,7 +13,9 @@
 	<key>pfm_ios_min</key>
 	<string>16.0</string>
 	<key>pfm_last_modified</key>
-	<date>2022-09-06T09:11:27Z</date>
+	<date>2022-12-14T14:18:42Z</date>
+	<key>pfm_macos_min</key>
+	<string>13.1</string>
 	<key>pfm_platforms</key>
 	<array>
 		<string>iOS</string>
```

Comment:
If we now have macOS preferences we need to also update the `pfm_platforms` to include it as well